samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AuthorFrontDoorMessages.properties

```diff
@@ -73,7 +73,7 @@ alt_sortLastModifiedAscending=Sort by Last Modified in Ascending
 alt_sortLastModifiedDescending=Sort by LastModified in Descending
 
 asterisk_2=**
-retracted_for_edit=To make a retracted assessment available to students, click "Edit" under Select Action, click "Edit" again, then select the appropriate republish option.
+retracted_for_edit=This assessment has been retracted. To make a retracted assessment available to students, click the "Actions" button and select "Edit", click "Edit" again, then select the appropriate republish option.
```

Comment:
"click Edit again" ?

Reply:
Makes sense to me, there are two edit buttons:
![edit](https://user-images.githubusercontent.com/8653754/80407583-23939c80-88c6-11ea-8775-7f5c626bb899.gif)

Reply:
how about `click the "Edit" button`  instead of `click "Edit" again`

Reply:
Sure, makes more sense, I just implemented the proposed text in the Jira.

Thanks Sam!